dll/win32/shimgvw/resource.h

```diff
@@ -51,11 +72,20 @@
 #define IDS_WMF_FILE    307
 
 /* Tooltips strings */
-#define IDS_TOOLTIP_NEXT_PIC   400
-#define IDS_TOOLTIP_PREV_PIC   401
-#define IDS_TOOLTIP_ZOOM_IN    402
-#define IDS_TOOLTIP_ZOOM_OUT   404
-#define IDS_TOOLTIP_ROT_CLOCKW 405
-#define IDS_TOOLTIP_ROT_COUNCW 406
-#define IDS_TOOLTIP_PRINT      407
-#define IDS_TOOLTIP_SAVEAS     408
+#define IDS_TOOLTIP_PREV        400
+#define IDS_TOOLTIP_NEXT        401
+#define IDS_TOOLTIP_BEST_FIT    402
+#define IDS_TOOLTIP_REAL_SIZE   403
+#define IDS_TOOLTIP_SLIDE_SHOW  404
+#define IDS_TOOLTIP_ZOOMP       405
+#define IDS_TOOLTIP_ZOOMM       406
+#define IDS_TOOLTIP_ROT1        407
+#define IDS_TOOLTIP_ROT2        408
```

Comment:
Any reason why these definitions were renamed? Their previous names are more readable.

Reply:
Example:

-> IDB_ROT1 for button image
-> IDC_ROT1 for button control identifier
-> TBICON_ROT1 for ImageList index

and...

-> IDS_TOOLTIP_ROT1 for string into tooltip.

Why? For example, look these two macros:

```
#define DECLARE_BTN_INFO(_name) \
    { TBICON_##_name, IDC_##_name, TBSTATE_ENABLED, BTNS_BUTTON, {0}, 0, 0 }
```
`#define DECLARE_BTN_CONFIG(_name) { IDB_##_name, IDS_TOOLTIP_##_name }
`

They made the source code simpler, more readable and easier to maintain, as you can see.
And, at logical level, I don't understand why we shouldn't use the same names for the various items...

Reply:
@carlo-bramini : I think @binarymaster 's comment was not about using "IDB/IDC/TBICON" prefixes, but instead why for example, you renamed ROT_CLOCKW into ROT1, ROT_COUNCW into ROT2, PREV_PIC into PREV, etc... everywhere.

Reply:
(OK I know that in the original file, some of the resource defines had the less readable names, but the comment was about why you chose not to use the more readable names consistently everywhere -- this would not have modified the macro name generating feature at all.)